app/src/main/java/org/qosp/notes/ui/notebooks/ManageNotebooksFragment.kt

```diff
@@ -145,4 +156,36 @@ class ManageNotebooksFragment : BaseFragment(R.layout.fragment_manage_notebooks)
             requireContext().resources.getDimension(R.dimen.app_bar_elevation)
         )
     }
+
+    private fun enlistNotebooks() {
+        // Get the current sort method preferences.
+        val sort = model.getSortNavdrawerNotebooksMethod()
+
+        activityModel.notebooks.collect(viewLifecycleOwner) { (_, notebooks) ->
+            // Apply sorting to the list of notebooks.
+            var sortedNotebooks: List<Notebook>
+            when (sort) {
```

Comment:
Same comment as the MainActivity

Reply:
Done. Carried out in f949e9c4118a72110ecfe55d7de3fbf63caaedbf.